.github/workflows/deploy.yml

```diff
@@ -10,8 +10,8 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 14.7
-      - run: yarn --frozen-lockfile
+          node-version: 14.x
+      - run: yarn --frozen-lockfilx
```

Comment:
Is this intended?

Reply:
This was a search-replace typo :see_no_evil: